### ISSUE TITLE
[BUG] Test 3.2.1 in CIS scan fails on hardened RKE2 cluster

### ISSUE THREAD

**Issue by Priyashetty17** [rancher/rancher#42171]
**Rancher Server Setup**
- Rancher version: 	v2.7-95f0b50ddf387c0d98a211f0217b34207348502d-head
- Installation option (Docker install/Helm Chart): Helm Chart
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): RKE2

**Information about the Cluster**
- Kubernetes version:  1.25.11, 1.26.6
- Cluster Type (Local/Downstream): Downstream
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider): Custom/Imported RKE2 (hardened, 1 node-all roles)

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom) Admin

**Describe the bug**
Test 3.2.1 in CIS scan fails on hardened RKE2 cluster

**To Reproduce**
1. Create a downstream hardened RKE2 cluster
2. Install the "rancher-cis-benchmark" on the downstream cluster (Chart version "4.2.0-rc2" and security tag "v0.2.13-rc3") 
3. Run CIS scan with profile "rke2-cis-1.7-profile-hardened"

**Result**
Test 3.2.1 in CIS scan fails
Note: This check passed with Chart 4.1.0 (cis-1.23 hardened profile) on the same cluster.

I see that the file does exist on the node.:
```
ubuntu@ip-172-31-20-22:~$ /bin/ps -ef | grep kube-apiserver | grep -v grep | grep -o audit-policy-file
audit-policy-file
```

Here's the snippet from the json report:
 ```
      "id": "3.2.1",
       "description": "Ensure that a minimal audit policy is created (Automated)",
       "remediation": "Create an audit policy file for your cluster.\n",
       "state": "fail",
       "node_type": [
         "node"
       ],
       "audit": "/bin/ps -ef | grep kube-apiserver | grep -v grep | grep -o audit-policy-file",
       "audit_config": "",
       "test_info": [
         "Create an audit policy file for your cluster.\n"
       ],
       "commands": null,
       "config_commands": null,
       "actual_value_per_node": {
         "ip-172-31-20-22": "audit-policy-file"
       },
       "expected_result": "'--audit-policy-file' is present",
       "test_type": "",
       "scored": true
     },
```

**Screenshots**
![image](https://github.com/rancher/rancher/assets/42880103/6c73bf30-9551-499d-8c0c-edb44413d20c)

CC @andypitcher @JonCrowther

**Comment by andypitcher** [comment]
@Priyashetty17  As per the recent [PR](https://github.com/rancher/security-scan/pull/150), could you run the scan again and test `3.2.1` against `rke2-cis-1.7-hardened ` with the last image version:

-  [rancher/security-scan:v0.2.13-rc4](https://hub.docker.com/layers/rancher/security-scan/v0.2.13-rc4/images/sha256-42ab456426f68ae959c69ecb7c97e7a8599342f893ce4ea7ce5905fc1dbca52f?context=explore) (DIGEST:sha256:42ab456426f68ae959c69ecb7c97e7a8599342f893ce4ea7ce5905fc1dbca52f)

Expected result  is a `PASS`.

**Comment by Priyashetty17** [comment]
@andypitcher Thanks for the fix! Test 3.2.1 check passed with security scan image [rancher/security-scan:v0.2.13-rc4](https://hub.docker.com/layers/rancher/security-scan/v0.2.13-rc4/images/sha256-42ab456426f68ae959c69ecb7c97e7a8599342f893ce4ea7ce5905fc1dbca52f?context=explore)

**Comment by andypitcher** [comment]
Thanks for confirming @Priyashetty17, I'm closing this issue.
